.circleci/config.yml

```diff
@@ -42,13 +48,18 @@ jobs:
     steps:
       - attach_workspace:
           at: /go
+      - restore_cache:
+          name: Restoring go mod cache
+          key: v1-gomod-{{ checksum "go.sum" }}
       - run:
           name: Install libseccomp
           command: |
             apt-get update -y
             apt-get install -y libseccomp-dev
       - run:
           name: Build source
+          environment:
+            GO111MODULE: "on"
```

Comment:
The other option, rather than setting `GO111MODULE` everywhere, is to simply move the source outside of `GOPATH`. Personal preference...